### PR TITLE
fork of pytest-jupyter v0.6.2:

### PR DESCRIPTION
**Pull request by akabanovs**
- removed noarch for python
 - updated script
 - updated dependencies
 - removed pytest code coverage due to it failing to reach pre-specified threshold
 - updated 'about' section

### CODE REVIEW

**Comment by akabanovs** [comment]
The recipe contains top level 'requirements' attribute, which currently lists 'host: python' and 'run: python' requirements.  Is this redundant, since the same is specified in each sub-package? i.e. should I remove it? 

Alternatively, can this top level specification be used to specify general host dependencies like: python,pip,wheel,setuptools, instead of defining them again and again in each sub-package?

**Comment by akabanovs** [comment]
the code coverage has been deactivated as the tests fail to reach the specified coverage of 90%. The coverage is 89.6%

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,31 +9,31 @@ source:
...
+        - hatchling 1.12.2
       run:
-        - python >=3.7
+        - python
         - jupyter_core
         - pytest
       run_constrained:
-        # the hard `run` dependencies are in sub-packages
         - jupyter_server >=1.21
         - jupyter_client >=7.4
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `jupyter_client >=7.4.0`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -50,16 +50,16 @@ outputs:
...
+        - python
+        - setuptools
+        - wheel
       run:
         - {{ pin_subpackage("pytest-jupyter", max_pin="x.x.x") }}
-        - ipykernel
-        - jupyter_client
-        - python >=3.7
+        - ipykernel >=6.14
+        - jupyter_client >=7.4
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: `jupyter_client >=7.4.0`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -9,31 +9,31 @@ source:
...
   number: 0
+  skip: true  # [py<37]
 
 requirements:
   host:
-    - python >=3.7
+    - python
   run:
-    - python >=3.7
+    - python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Without `run` it can work too

Comment (akabanovs) [reply recipe/meta.yaml]: But do we need this whole top level requirements section at all, since we specify python requirement in each sub-package?

Comment (skupr-anaconda) [reply recipe/meta.yaml]: It will fail randomly without requirements/host (it's a conda-build bug, IMO). But requirements/run is redundant.

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml]: done

File: recipe/meta.yaml (hunk 2 above)
Comment (akabanovs) [reply recipe/meta.yaml]: done

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 3 above)
Comment (akabanovs) [reply recipe/meta.yaml]: done.

**Pull request merged**